examples/model_compress/pruning/auto_pruners_torch.py

```diff
@@ -1,7 +1,9 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
+
 '''
-Examples for automatic pruners
+Example for supported automatic pruning algorithms.
+In this example, we present the usage of automatic pruners (NetAdapt, AutoCompressPruner). L1, L2, FPGM pruners are aims for comparsion.
```

Comment:
"are aims for comparison" -> "are also executed for comparison purpose"

Reply:
Thanks, fix it.